concepts/booleans/about.md

```diff
@@ -0,0 +1,15 @@
+# About
+
+The boolean type `Bool` is an enumeration of `True` and `False`. The basic boolean operators are `&&` (and), `||` (or), and `not`.
+
+```haskell
+True || False           -- True
+True && False           -- False
+```
+
+The three boolean operators each have a different operator precedence. They are evaluated in this order: `not` first, `&&` second, and finally `||`. If you want to bypass these rules, you can enclose a boolean expression in parentheses (`()`), as the parentheses have an even higher operator precedence.
```

Comment:
I would say "override" rather than "bypass" in this case.

A bypass is a road that passes around something, such as a residential area while an override is a mechanism, device or procedure used to counteract an automatic control.

Reply:
Edited introduction.md and about.md to reflect suggested change.